### PR TITLE
fix

### PR DESCRIPTION
**Pull request by lieblichaaron**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

https://user-images.githubusercontent.com/48128556/180860283-6d5a8924-e760-4dab-b2c8-9ee86484cb3c.mov

### CODE REVIEW

**Review by avivbarniv — COMMENTED** [review]
Can u add a video?

File: examples/client/Locomotion/src/pages/ActiveRide/newMap.js [hunk 1]
```diff
@@ -221,18 +230,14 @@ export default React.forwardRef(({
         key="map"
         followsUserLocation={isUserLocationFocused}
         moveOnMarkerPress={false}
-        onRegionChangeComplete={async (event) => {
+        onPanDrag={({ nativeEvent: { coordinate: { latitude, longitude } } }) => {
+          if (!isUserLocationFocused === false) {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/ActiveRide/newMap.js]: Why not isUserLocationFocused?

Comment (lieblichaaron) [reply examples/client/Locomotion/src/pages/ActiveRide/newMap.js]: thats was already there

Comment (avivbarniv) [reply examples/client/Locomotion/src/pages/ActiveRide/newMap.js]: it doesn't mean you don't need to change it :)

Comment (lieblichaaron) [reply examples/client/Locomotion/src/pages/ActiveRide/newMap.js]: i just didnt know if it was being used for something?

**Pull request merged**